pyiron_base/jobs/job/generic.py

```diff
@@ -632,10 +633,15 @@ def write_input(self):
         Call routines that generate the code specific input files
         Returns:
         """
-        write_input_files_from_input_dict(
-            input_dict=self.get_input_parameter_dict(),
-            working_directory=self.working_directory,
-        )
+        if not self._input_is_written:
```

Comment:
@Leimeroth Is there an issue with files being written twice? Meaning the remote job trying to write the files again and in the worst case overwriting the existing input files.

Reply:
I think the write_input_funct is called again somewhere on the cluster. This call tries to copy the potential file, 
so I get the errors when using my own potentials.